packages/core/src/browser/shell/shell-layout-restorer.ts

```diff
@@ -109,16 +109,17 @@ export interface ApplicationShellLayoutMigration {
     onWillInflateWidget?(desc: WidgetDescription, context: ApplicationShellLayoutMigrationContext): MaybePromise<WidgetDescription | undefined>;
 }
 
-const resetLayoutCommand = {
+export const RESET_LAYOUT: Command = {
```

Comment:
[minor] do we need a `changelog` note about this command?

Reply:
No, it is not a breaking API change.

Reply:
But you are definitely right about the changelog in general; switching from `unload` to `beforeunload` is a breaking change. I do an update. Thanks for the reminder 👍

Reply:
Here: https://github.com/eclipse-theia/theia/pull/6532/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR5

Reply:
@kittaakos thank you and no problem :)